Prefabs/Player.prefab

```diff
@@ -530,12 +530,13 @@
                                     "ParticleAsset": {
                                         "guid": "{123FCBB4-016F-56D6-9DEC-BD329101C46F}"
                                     },
-                                    "AudioTrigger": "Play_Sound_Gun_Firing"
+                                    "AudioTrigger": "play_sx_wpn_laserpistol_fire"
                                 },
                                 "ImpactFx": {
                                     "ParticleAsset": {
                                         "guid": "{1BB94CA2-0BA0-5EC6-BE46-9B1F51D66CB9}"
-                                    }
+                                    },
+                                    "AudioTrigger": "play_sx_wpn_laserpistol_impact"
```

Comment:
This will double-play the sound.. While the impact and damage effects overlay nicely, we probably only want the actual sound effect on either the impact or the damage. I put it on the damage so there's more feedback about what the server did.